python/dgl/data/citation_graph.py

```diff
@@ -11,10 +11,17 @@
 import scipy.sparse as sp
 import os, sys
 
-from .utils import download, extract_archive, get_download_dir, _get_dgl_url
+from .dgl_dataset import DGLBuiltinDataset
+from .utils import download, extract_archive, get_download_dir
+from .utils import save_graphs, load_graphs, save_info, load_info, makedirs, _get_dgl_url
+from .utils import generate_mask_tensor
 from ..utils import retry_method_with_fix
+from .. import backend as F
 from ..graph import DGLGraph
 from ..graph import batch as graph_batch
+from ..convert import to_networkx
+
+backend = os.environ.get('DGLBACKEND', 'pytorch')
 
 _urls = {
```

Comment:
can we make _urls as a class attribute inside CitationGraphDataset?

Reply:
yes